runtime/src/genesis_utils.rs

```diff
@@ -200,6 +200,18 @@ pub fn activate_all_features(genesis_config: &mut GenesisConfig) {
     }
 }
 
+pub fn activate_all_features_except(
+    genesis_config: &mut GenesisConfig,
+    features: &HashSet<Pubkey>,
+) {
+    // Activate all features at genesis in development mode except for those provided in "features" hashset
+    for feature_id in FeatureSet::default().inactive {
+        if !features.contains(&feature_id) {
+            activate_feature(genesis_config, feature_id);
+        }
+    }
```

Comment:
I was looking to see how test-validator currently accomplishes the deactivation; you can see it here:
https://github.com/anza-xyz/agave/blob/923355cbfd13c0bf0ca26db93a2649086e2ef7b9/test-validator/src/lib.rs#L830-L846

Basically, all features are previously activated, and then deactivate (remove) the user set deactivate features. I'm not sure that what test-validator does is better than what you did, but noted here for completeness. What do you think ?

Reply:
it is a little weird to activate them all and then deactivate specific ones. But for consistency I think we change it to just that. Activate all, then deactivate specific ones pass in

Reply:
Agreed about it being a little weird, but I also like the consistency between the two so